plugins/scaffolder-backend/api-report.md

```diff
@@ -287,6 +290,11 @@ export function fetchContents({
   outputPath: string;
 }): Promise<void>;
 
+// Warning: (ae-missing-release-tag) "NunjucksFilter" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export type NunjucksFilter = (...args: JsonValue[]) => JsonValue | undefined;
```

Comment:
So with the rename to `additionalTemplateFilters`, I think we've gotta call this `TemplateFilter`

Reply:
Updated.

Reply:
Marking this resolved.